dtreeviz/forests.py

```diff
@@ -0,0 +1,246 @@
+import numpy as np
+import pandas as pd
+
+import matplotlib.patches as patches
+import matplotlib.pyplot as plt
+from matplotlib.collections import PatchCollection
+from colour import Color
+from PIL import ImageColor
+
+from dtreeviz.colors import adjust_colors, GREY
+from dtreeviz.trees import ctreeviz_bivar, add_classifier_legend
+from dtreeviz.models.shadow_decision_tree import ShadowDecTree
+
+
+def ctreeviz_bivar_fusion(trees, X:np.ndarray, y:np.ndarray,
+                          feature_names, target_name, class_names=None,
+                          fontsize=12,
+                          fontname="Arial",
+                          show_region_edges=True,
+                          alpha=.1,
+                          ax=None):
+    if ax is None:
+        fig, ax = plt.subplots(1, 1, figsize=(4, 3.5))
+    ax.spines['top'].set_visible(False)  # turns off the top "spine" completely
+    ax.spines['right'].set_visible(False)
+    ax.spines['left'].set_linewidth(.5)
+    ax.spines['bottom'].set_linewidth(.5)
+
+    class_names = ShadowDecTree.normalize_class_names(class_names, nclasses=len(np.unique(y)))
```

Comment:
since normalize_class_names() is used both for trees and random forests, would be an idea to extract this method in a util module ?

Reply:
that's a good idea. I'll move it.

Reply:
ShadowDecTree.normalize_class_names is not defined anymore :)

Reply:
@parrt I can solve this issues if you dont have time ?

Reply:
true, we have created and put the normalize_class_names in utils module, but there where still few old references like ShadowDecTree.normalize_class_names. 

Created a PR to fix it.